src/Symfony/Component/Mailer/Bridge/Mailtrap/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "symfony/mailtrap-mailer",
+    "type": "symfony-mailer-bridge",
+    "description": "Symfony Mailtrap Mailer Bridge",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "psr/event-dispatcher": "^1",
+        "symfony/mailer": "^6.4|^7.0"
+    },
+    "require-dev": {
+        "symfony/http-client": "^6.4|^7.0",
+        "symfony/webhook": "^6.4|^7.0"
+    },
+    "conflict": {
+        "symfony/http-foundation": "<6.4"
+    },
```

Comment:
can you confirm this is needed? most of the time, those are copy/paste things from other bridges 😄